jobs/redis-blacksmith-plans/templates/plans/standalone/manifest.yml

```diff
@@ -36,3 +36,12 @@ instance_groups:
           auth:
             password: (( grab meta.password ))
           persistent: (( grab meta.persistent ))
+          redis:
+            maxmemory-policy: (( grab meta.params.maxmemory-policy || "allkeys-lru" ))
+            notify-keyspace-events: (( grab meta.params.notify-keyspace-events || "" ))
+            slowlog-log-slower-than: (( grab meta.params.slowlog-log-slower-than || 10000 ))
+            slowlog-max-len: (( grab meta.params.slowlog-max-len || 128 ))
+          client:
+            timeout: (( grab client.timeout || 0 ))
+            tcpkeepalive: (( grab meta.client.tcpkeepalive || 300 ))
+            connections: (( grab meta.client.connections || 10000 ))
```

Comment:
Same as with the other redis manifest; either these should all be meta.client or none should be.

Reply:
Fixing to meta.client